src/main/java/school/lemon/changerequest/java/introduction/hw2/Task10.java

```diff
@@ -1,7 +1,19 @@
 package school.lemon.changerequest.java.introduction.hw2;
 
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 public class Task10 {
     public static String trim(String text) {
-        return "";
+        if (text == null) {
+            return null;
+        }
+        Pattern p = Pattern.compile("(((?:=[\\t]\\s)[a-zA-z]+.*[^\\s\\t]+)|([a-zA-Z]).*[^\\t\\s])");
```

Comment:
1. Why do you need `\\t` separately from `\\s`? It's also included in `\\s`
2. Not sure about only `a-zA-Z` group. How about digits and other non-space characters?
3. Why do you use exclusion there `[^\\t\\s]`? It can be done simply using `\\S`.
4. In general it can be simplified.